src/sentry/static/sentry/app/views/groupDetails/shared/groupDetails.jsx

```diff
@@ -78,6 +80,11 @@ const GroupDetails = createReactClass({
     if (this.props.environment) {
       query.environment = this.props.environment.name;
     }
+    if (this.props.organization) {
+      query.enable_snuba = (this.props.organization.features || []).includes('sentry10')
```

Comment:
This is effectively a double check since `organization` is only passed in the Sentry 10 variant